src/model/ScheduleTypeRegistry.cpp

```diff
@@ -236,6 +236,9 @@ namespace model {
       {"DefaultScheduleSet", "Steam Equipment", "steamEquipmentSchedule", true, "", 0.0, 1.0},
       {"DefaultScheduleSet", "Other Equipment", "otherEquipmentSchedule", true, "", OptionalDouble(), OptionalDouble()},
       {"DesignSpecificationOutdoorAir", "Outdoor Air Flow Rate", "outdoorAirFlowRateFractionSchedule", true, "", 0.0, 1.0},
+      {"DistrictCooling", "Capacity Fraction", "capacityFractionSchedule", true, "Dimensionless", 0.0, 1.0},
+      {"DistrictHeating", "Capacity Fraction", "capacityFractionSchedule", true, "Dimensionless", 0.0, 1.0},
+      {"DistrictHeatingSteam", "Capacity Fraction", "capacityFractionSchedule", true, "Dimensionless", 0.0, 1.0},
```

Comment:
E+ allows you to input a schedule that has > 1.

https://github.com/NREL/EnergyPlus/blob/281383d95af45331e9c1fba41fa1dba5d66a59c5/src/EnergyPlus/OutsideEnergySources.cc#L284-L295

The above only checks for a minimum.


I suggest using `0.0, OptionalDouble()`